docs(revamp): Fix main and multi-cloud README.md, revamp site (#1489)
microsoft/retina: docs/02-Installation/02-CLI.md

# CLI Setup

> Retina CLI allows to run captures which are distributed packet captures leveraging [tcpdump](https://www.tcpdump.org/). Currently, Retina CLI supports Linux, Windows, and MacOS on x86_64 and ARM64 platforms. For CLI usage, see [Capture with Retina CLI](../04-Captures/02-cli.md).

Reviewer: ```suggestion
Retina CLI allows you to run distributed packet captures leveraging [tcpdump](https://www.tcpdump.org/). Currently, Retina CLI supports Linux, Windows, and MacOS on x86_64 and ARM64 platforms. For CLI usage, see [Capture with Retina CLI](../04-Captures/02-cli.md).
```
